src/server/utils/bucket_diff.js

```diff
@@ -69,38 +69,51 @@ class BucketDiff {
             //     ]
             // }
             keys_diff_map: {},
+            keys_del_arr: [],
```

Comment:
I was re thinking, and i think we should use an object here instead of an array. We should carry the delete key with all it's properties as we do in the versions diff.

Reply:
sure, will update it. Thanks